imageroot/actions/create-module/10initialize

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+#
+# Copyright (C) 2022 Nethesis S.r.l.
+# SPDX-License-Identifier: GPL-3.0-or-later
+#
+
+set -e
+
+exec 1>&2 # Send any output to stderr, to not alter the action response protocol
+
+install -m 644 "${AGENT_INSTALL_DIR}/crowdsec.service" "/etc/systemd/system/${MODULE_ID}.service"
+systemctl daemon-reload
+systemctl enable --now "${MODULE_ID}.service"
+
+
+# we need to start again to expand configuration
+systemctl restart "${MODULE_ID}.service"
```

Comment:
Uh that's ugly.
It's not a real problem starting it twice, but I'd ask inside the crowdsec community to see if it's a normal behavior.

Reply:
@gsanchietti check the script 

https://github.com/crowdsecurity/crowdsec/blob/master/docker/docker_start.sh#L25

We erase all previous configuration done at the first start

Reply:
If we do not need the configuration from `staging` dir, we could create an empty `/etc/crowdsec/config.yaml` file to avoid the deletion.
What do you think?

Reply:
done